File: Sources/ATTNOrder.m
//
//  ATTNOrder.m
//  attentive-ios-sdk
//
//  Created by Wyatt Davis on 12/16/22.
//

#import "ATTNOrder.h"

NS_ASSUME_NONNULL_BEGIN

@implementation ATTNOrder

- (instancetype)initWithOrderId:(NSString *)orderId {
  if (self = [super init]) {
    self->_orderId = orderId;
  }

  return self;
}

@end

NS_ASSUME_NONNULL_END

File: Sources/Public/Events/ATTNOrder.swift
//
//  ATTNOrder.swift
//  attentive-ios-sdk-framework
//
//  Created by Vladimir - Work on 2024-05-30.
//

import Foundation

@objc(ATTNOrder)
public final class ATTNOrder: NSObject {
  @objc public var orderId: String

  @objc(initWithOrderId:)
  public init(orderId: String) {
    self.orderId = orderId
  }

  private override init() {
    fatalError("init() has not been implemented")
  }
}
